### PR TITLE
Dark Bootstrap theme, view JSON output in UI

### PR DESCRIPTION
**Pull request by buchowski**

### CODE REVIEW

File: ui/app/css/bootstrap-superhero.min.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
... .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle),.input-group-btn:last-child>.btn-group:not(:last-child)>.btn{border-bottom-right-radius:0;border-top-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:first-child>.btn-group:not(:first-child)>.btn{border-bottom-left-radius:0;border-top-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:hover,.input-group-btn>.btn:focus,.input-group-btn>.btn:active{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-grou
```
Comment (MichaelSitter) [inline ui/app/css/bootstrap-superhero.min.css]: Is this supposed to be tracked? Is there an npm package available?

**Comment by MichaelSitter** [comment]
:eagle:

File: ui/app/css/bootstrap-superhero.min.css (hunk 1 above)
Comment (relistan) [inline ui/app/css/bootstrap-superhero.min.css]: There are no npm packages that I know of.

**Comment by relistan** [comment]
Nice @buchowski . Looks like the logo doesn't look so good on a dark background. Maybe we drop it off there for now. Two minor nits: 
1. I think we'll want it to require a click to see the JSON. As I scroll over the page it pops up unexpectedly when I roll over the `(i)`. 
2. The Details buttons are huge when using the Boostrap buttons instead of native buttons. I am not sure what the best thing is to do there, but the line of text is actually less prominent now than the button. Is it easy to shrink the button? I didn't figure it out before.

Finally, any chance you can force push this branch again with the Nitro internal Jira tickets filtered from the commits? Sorry man, thanks!

**Pull request closed without merging**